package.json

```diff
@@ -58,13 +68,15 @@
     "browserstack": "grunt build && sh build/run-browserstack.sh",
     "build": "grunt build",
     "test": "grunt",
-    "coverage": "grunt coverage"
+    "coverage": "grunt coverage",
+    "test:cli": "grunt build && npm link && qunit test/cli/*.js"
```

Comment:
I'm a noob on npm features, this will call `qunit` set in the `bin` property in this same file, here?